packages/build-info/src/node/file-system.ts

```diff
@@ -23,6 +27,14 @@ export class NodeFS extends FileSystem {
     return relative(from, to)
   }
 
+  basename(path: string): string {
+    return basename(path)
+  }
+
+  join(...segments: string[]): string {
+    return join(...segments)
+  }
```

Comment:
Just noting that path is [easy to polyfill in the browser](https://webpack.js.org/configuration/resolve/#resolvefallback), so we do not necessarily need to add it to the FileSystem here, but I'm also not against doing that.

Reply:
But then you would need webpack ;) and I've built an implementation for it (added good tests as well)